torchvision/transforms/functional_tensor.py

```diff
@@ -702,9 +703,6 @@ def pad(img: Tensor, padding: List[int], fill: int = 0, padding_mode: str = "con
     Returns:
         Tensor: Padded image.
     """
-    if not _is_tensor_a_torch_image(img):
```

Comment:
nit: just saw that you removed this now, any particular reason?

Reply:
Yes. `lut` on `_scale_channel()` is 1-dimensional so to reuse `pad()` we need to lose this assertion.

Reply:
restored